### ISSUE TITLE
Port 8000 cannot be accessed via a svc on minikube

### ISSUE THREAD

**Issue by skonto** [ceph/cn#115]
I was using ceph/daemon latest tag following `cn kube` for the integration tests [here](https://github.com/apache/spark/pull/23546). The idea is to access the 8000 port with a svc created on minikube and which can be accessed on the host machine outside the minikube vm. It used to work but the recent update has broken the access to the container's port for some reason (http://192.168.39.193:32194).

```
./minikube service list
|-------------|--------------|-----------------------------|
|  NAMESPACE  |     NAME     |             URL             |
|-------------|--------------|-----------------------------|
| default     | kubernetes   | No node port                |
| kube-system | kube-dns     | No node port                |
| spark       | ceph-nano-s3 | http://192.168.39.193:32194 |
|-------------|--------------|-----------------------------|

```
```
$ ./kubectl get svc -n spark
NAME           TYPE       CLUSTER-IP      EXTERNAL-IP   PORT(S)          AGE
ceph-nano-s3   NodePort   10.98.131.126   <none>        8000:32194/TCP   11m
```

Minikube is running with kvm2. This used to work with `latest` and now I am falling back to  `ceph/daemon:v4.0.0-stable-4.0-master-centos-7-x86_64`. Maybe I should use the stable version thoughts?

**Comment by deep-j** [comment]
Adding 
```
 -name: RGW_FRONTEND_PORT
  value: "8000" 
```
to the env section in the Kubernetes template will resolve the port issue with the `latest` image.

**Comment by skonto** [comment]
Will try it and see.

**Comment by leseb** [comment]
Closed via #118
